src/net/WebRTC/RTCPeerConnection.cs

```diff
@@ -218,10 +218,29 @@ public RTCCertificate CurrentCertificate
         /// </summary>
         public event Action onnegotiationneeded;
 
+        private event Action<RTCIceCandidate> _onIceCandidate;

```

Comment:
I think this approach is an improvement on my original one. Chrome does the same thing, I wonder if there will be side effects to putting the original host candidates in the SDP and also supplying them in the event. It should be fine.